test-components/rust-service/rpc/Cargo.toml

```diff
@@ -0,0 +1,3 @@
+[workspace]
```

Comment:
Is there a reason for putting this _inside_ of the already existing `rust-service` test component? I find it confusing as all the other test components are directly in subdirectories of `test-components`